distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -83,18 +82,18 @@ type Any
         self_meta = Meta.meta self
         that_meta = Meta.meta that
         case Pair.new self_meta that_meta of
-            Pair.Value (Meta.Atom_Data _) (Meta.Atom_Data _) ->
+            Pair.Value (Meta.Atom.Atom_Data _) (Meta.Atom.Atom_Data _) ->
```

Comment:
Is this a simplification? Looks longer than before...

Reply:
Yes I missed those 4 `_Data`. Spotted late last night and just testing so is `Meta.Atom.Value` now.